### DISCUSSION TITLE
Seed Number Changing

### DISCUSSION THREAD

**Discussion by rmarkjr** [lllyasviel/Fooocus#2177] [Q&A] [answered]
Hello! I have noticed that when I unselect the seed and use a specific seed value that the generated images will still produce a different seed number. Of course the characters change a bit. Now I think I see why. 

* I am trying to make a consistent character. I have the same seed number, image input with Faceswap, etc 

If I generate 1 image that seed number will be the same.

However, if I produce 4 images the seed numbers will each be different by one number, I think in a sequential order like 6324, 6325, 6326, 6327 etc. 

Is this normal or to be expeced? If so, then why have an option to set a specific seed if it will still generate different seed? Is there a way to actually lock that to one seed and generate multiple images?

Thank you!

**Comment by mashb1t** [comment, marked as answer]
Yes, just answered that 3 days ago: https://github.com/lllyasviel/Fooocus/issues/2141

**Reply by rmarkjr** [reply]
ah damn. Sorry I didn't see it on the similar topic search thing

**Comment by poor7** [comment]
If you create 4 images and prompt and seed do not change in any way, then the 4 created images will be identical (the same).

**Reply by rmarkjr** [reply]
ya thats what I kinda thought after I posted it. I was 50/50 on either a new seed for each to be a little different or the same seed to look the same but change with different prompts
